cdap-explore/src/main/java/co/cask/cdap/hive/serde/ObjectSerializer.java

```diff
@@ -72,7 +73,7 @@ public Writable serialize(Object o, ObjectInspector objectInspector) {
     List<TypeInfo> info = structTypeInfo.getAllStructFieldTypeInfos();
     List<String> names = structTypeInfo.getAllStructFieldNames();
 
-    Map<String, Object> recordMap = Maps.newConcurrentMap();
+    Map<String, Object> recordMap = new HashMap<>();
```

Comment:
don't know why it was before.  It is only used in this one method where it is serialized to json.  In fact, it can't be a concurrentMap because we need to support null values.